src/script/components/Conversation/JumpToLastMessageButton.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {FC, HTMLProps} from 'react';
+
+// eslint-disable-next-line import/no-unresolved
```

Comment:
without this type, there will be test failure due to:
```
   Error: src/script/components/Conversation/JumpToLastMessageButton.tsx(45,6): error TS2322: Type '{ children: Element; accept?: string; acceptCharset?: string; action?: string; allowFullScreen?: boolean; allowTransparency?: boolean; alt?: string; as?: string; async?: boolean; autoComplete?: string; ... 358 more ...; css: { ...; }; }' is not assignable to type 'IconButtonProps<HTMLButtonElement>'.
  Types of property 'ref' are incompatible.
    Type 'LegacyRef<HTMLElement>' is not assignable to type 'Ref<HTMLButtonElement>'.
      Type 'string' is not assignable to type 'Ref<HTMLButtonElement>'.
```

to fix this ESLint issue, StackOverflow recommends to add 
```
extends: [
    "plugin:import/errors",
    "plugin:import/warnings",
    "plugin:import/typescript",
  ],
```
but it doesn't help (and we already have some of them in the `@wireapp/eslint-config`

Without using `{...rest}` the CSS will not get applied on `IconButton`, and setting the position CSS inside of the `JumpToLastMessageButton` looks as a bad solution, it should be positioned where it is used

Reply:
seems like a typing issue

Reply:
i think this solution is a smell. i would prefer the css inside of the component and just pass the variable in and decide the size based on var

Reply:
yes, it is a type issue, and using `WithConditionalCSSProp` solves it. but this import causes the `ESLint` error

this is the CSS positioning we have to set:
```
  position: 'absolute',
  bottom: mdBreakpoint? '100px' : '56px',
  right: '10px',
  height: '40px',
```
Do you mean you'd prefer to have a separate custom `CSSObject` prop in `JumpToLastMessageButtonProps` instead of `css`  and apply it on `IconButton`?

Reply:
we decided to leave this component inside of the `MessageList` file, as it will only be used there, so that we can hardcode all the styles inside of it without any `...rest` or other props way logic

Reply:
done